### ISSUE TITLE
Default NTP server name

### ISSUE THREAD

**Issue by jeremypoulter** [OpenEVSE/openevse_esp32_firmware#273]
We shouldn't really be using  `pool.ntp.org` as a default NTP server (see <https://www.ntppool.org/en/vendors.html>), we should [apply](https://manage.ntppool.org/manage/vendor) for something like `openevse.pool.ntp.org` and set that as the default.

**Comment by jeremypoulter** [comment]
@chris1howell would you be able to look at setting up this?

**Comment by ausil** [comment]
+1 for this

**Comment by gmaxwell** [comment]
It would still be good to take the dhcp provided NTP server!

**Comment by RAR** [comment]
The DHCP part of this thread is implemented in #1289 (firmware: prefer the NTP server from DHCP option 42, fall back to the configured host if it stops answering, `sntp_dhcp` toggle defaulting on) with the UI in OpenEVSE/openevse-gui-nightshift#143. It needs a test on a network whose DHCP server hands out option 42 — checklist in the PR. The original ask here (an `openevse.pool.ntp.org` vendor zone as the default) is an admin step at manage.ntppool.org plus a one-line change to `SNTP_DEFAULT_HOST`, still open.

**Comment by RAR** [comment]
And for the original ask: #1290 switches the fresh-install default to `time.cloudflare.com` (free for any client, no vendor registration to maintain) rather than applying for a pool.ntp.org vendor zone; GUI placeholders in OpenEVSE/openevse-gui-nightshift#144. Existing installs keep their stored `sntp_hostname`.

**Comment by jeremypoulter** [comment]
Just to mirror my comment on the PR this ticket was about doing the registration, rather than switching to another default.

**Comment by RAR** [comment]
Totally - but after 5 years that hasn't happened, we're still using pool.ntp.org which they ask us not to.   We can leave it and close this out - the NTP via DHCP ask is under https://github.com/OpenEVSE/openevse_esp32_firmware/pull/1289.  

Just don't want this open 5 years from now collecting dust.

**Comment by jeremypoulter** [comment]
Agreed @chris1howell is this something you can look at? Either register with ntp.org or accept the PR?

**Comment by RAR** [comment]
> It would still be good to take the dhcp provided NTP server!

<img width="704" height="1031" alt="Image" src="https://github.com/user-attachments/assets/c30bec34-9db0-4b06-a312-31c67d063772" />

will be in next release.
